src/main/java/com/stripe/model/Account.java

```diff
@@ -196,40 +202,6 @@ public Account update(Map<String, Object> params, RequestOptions options)
 		return request(RequestMethod.POST, instanceURL(Account.class, this.id), params, Account.class, options);
 	}
 
-	@Override
-	public boolean equals(Object o) {
```

Comment:
nevrermind: didn't realize that we tend not to have equals methods.  Sounds like that's for fixing another day.